tests/sentry/rules/processing/test_delayed_processing.py

```diff
@@ -681,17 +702,18 @@ def test_apply_delayed_percent_comparison_condition_interval(self):
         the correct comparison interval, e.g. # events is ... compared to 1 hr ago
         """
         percent_condition = self.create_event_frequency_condition(
-            interval="5m",
+            interval="1h",
```

Comment:
The interval and comparison interval need to be changed for this test in order to steer clear of any snuba rollup issues. The rollup for intervals of <= 1 hr is only 10 seconds, so it's effect is minimal on shifting the query boundaries